<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BlastDialog</class>
 <widget class="QDialog" name="BlastDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>320</width>
    <height>456</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>320</width>
    <height>380</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>320</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>BLAST sequence(s)</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../../resources.qrc">
    <normaloff>:/aliases/images/icons/blast</normaloff>:/aliases/images/icons/blast</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="margin">
    <number>4</number>
   </property>
   <item>
    <layout class="QGridLayout" name="gridLayout" columnstretch="0,0,1,1">
     <property name="sizeConstraint">
      <enum>QLayout::SetMaximumSize</enum>
     </property>
     <item row="2" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>BLAST Database:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
       <property name="buddy">
        <cstring>blastDatabaseComboBox</cstring>
       </property>
      </widget>
     </item>
     <item row="2" column="1" colspan="3">
      <widget class="QComboBox" name="blastDatabaseComboBox"/>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>E-value threshold:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
       <property name="buddy">
        <cstring>evalueLineEdit</cstring>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="evalueLineEdit">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>70</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>.01</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>Max hits:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
       <property name="buddy">
        <cstring>maxHitsSpinBox</cstring>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QSpinBox" name="maxHitsSpinBox">
       <property name="maximumSize">
        <size>
         <width>70</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>10000</number>
       </property>
       <property name="value">
        <number>250</number>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Matrix:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
       <property name="buddy">
        <cstring>matrixComboBox</cstring>
       </property>
      </widget>
     </item>
     <item row="5" column="1" colspan="2">
      <widget class="QComboBox" name="matrixComboBox">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="maximumSize">
        <size>
         <width>125</width>
         <height>16777215</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="7" column="0">
      <widget class="QCheckBox" name="psiBlastCheckBox">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="font">
        <font>
         <weight>50</weight>
         <italic>true</italic>
         <bold>false</bold>
        </font>
       </property>
       <property name="text">
        <string>PSI-BLAST</string>
       </property>
       <property name="checked">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="6" column="0" colspan="4">
      <widget class="Line" name="line_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="11" column="0">
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string>Threads:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
       <property name="buddy">
        <cstring>threadsSpinBox</cstring>
       </property>
      </widget>
     </item>
     <item row="10" column="0" colspan="4">
      <widget class="Line" name="line_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="11" column="2" colspan="2">
      <widget class="QSlider" name="threadsSlider">
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>99</number>
       </property>
       <property name="pageStep">
        <number>1</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="tickInterval">
        <number>1</number>
       </property>
      </widget>
     </item>
     <item row="12" column="0" colspan="4">
      <widget class="Line" name="line">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="13" column="0" colspan="4">
      <widget class="QDialogButtonBox" name="buttonBox">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="standardButtons">
        <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
       </property>
      </widget>
     </item>
     <item row="1" column="0" colspan="4">
      <widget class="QTableWidget" name="sequenceTableWidget">
       <property name="editTriggers">
        <set>QAbstractItemView::NoEditTriggers</set>
       </property>
       <property name="alternatingRowColors">
        <bool>true</bool>
       </property>
       <property name="selectionBehavior">
        <enum>QAbstractItemView::SelectRows</enum>
       </property>
       <property name="wordWrap">
        <bool>false</bool>
       </property>
       <attribute name="horizontalHeaderStretchLastSection">
        <bool>true</bool>
       </attribute>
       <attribute name="verticalHeaderDefaultSectionSize">
        <number>22</number>
       </attribute>
       <column>
        <property name="text">
         <string>Name</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Sequence</string>
        </property>
       </column>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Sequences:</string>
       </property>
       <property name="buddy">
        <cstring>sequenceTableWidget</cstring>
       </property>
      </widget>
     </item>
     <item row="5" column="3">
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>98</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="3" column="2" colspan="2">
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>75</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="4" column="2" colspan="2">
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="11" column="1">
      <widget class="QSpinBox" name="threadsSpinBox">
       <property name="maximumSize">
        <size>
         <width>70</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="minimum">
        <number>1</number>
       </property>
      </widget>
     </item>
     <item row="9" column="1">
      <widget class="QLineEdit" name="inclusionLineEdit">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="maximumSize">
        <size>
         <width>70</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>.002</string>
       </property>
      </widget>
     </item>
     <item row="9" column="0">
      <widget class="QLabel" name="label_8">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Inclusion E-value:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
       <property name="buddy">
        <cstring>inclusionLineEdit</cstring>
       </property>
      </widget>
     </item>
     <item row="8" column="0">
      <widget class="QLabel" name="label_6">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Iterations:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
       <property name="buddy">
        <cstring>iterationsSpinBox</cstring>
       </property>
      </widget>
     </item>
     <item row="8" column="1">
      <widget class="QSpinBox" name="iterationsSpinBox">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="maximumSize">
        <size>
         <width>70</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="minimum">
        <number>2</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../resources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>BlastDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>209</x>
     <y>450</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>BlastDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>209</x>
     <y>450</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>threadsSpinBox</sender>
   <signal>valueChanged(int)</signal>
   <receiver>threadsSlider</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>203</x>
     <y>408</y>
    </hint>
    <hint type="destinationlabel">
     <x>314</x>
     <y>404</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>threadsSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>threadsSpinBox</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>314</x>
     <y>404</y>
    </hint>
    <hint type="destinationlabel">
     <x>203</x>
     <y>408</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>psiBlastCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>label_6</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>46</x>
     <y>294</y>
    </hint>
    <hint type="destinationlabel">
     <x>77</x>
     <y>316</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>psiBlastCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>iterationsSpinBox</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>109</x>
     <y>290</y>
    </hint>
    <hint type="destinationlabel">
     <x>159</x>
     <y>312</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>psiBlastCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>label_8</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>28</x>
     <y>292</y>
    </hint>
    <hint type="destinationlabel">
     <x>48</x>
     <y>350</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>psiBlastCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>inclusionLineEdit</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>27</x>
     <y>298</y>
    </hint>
    <hint type="destinationlabel">
     <x>171</x>
     <y>343</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
